WordPress/gutenberg
# Adding Font size presets UI

## What?
Adds UI to Global styles to allow modification of font size presets.
Fixes: https://github.com/WordPress/gutenberg/issues/61987

This implementation started here https://github.com/WordPress/gutenberg/pull/62328 but the approach used changed so I'm continuing that work in this new PR. 

## Why?
To allow users to edit the font size presets using the editor.

## How?
By implementing the UI to handling creation, update and removal of font size presets.

## Testing Instructions
Watch the screencast to see how the UI works and try to use the feature on your testing env.

## Screenshots or screencast 
[Screencast from 02-07-24 17:45:39.webm](https://github.com/WordPress/gutenberg/assets/1310626/a952f941-b8a0-4e48-aa57-8f1c5a4b0716)

## :memo: Reminder:
Before merging add the props from the first PR:
```
Co-authored-by: matiasbenedetto <mmaattiiaass@git.wordpress.org>
Co-authored-by: t-hamano <wildworks@git.wordpress.org>
Co-authored-by: tyxla <tyxla@git.wordpress.org>
Co-authored-by: jasmussen <joen@git.wordpress.org>
Co-authored-by: mirka <0mirka00@git.wordpress.org>
Co-authored-by: ciampo <mciampini@git.wordpress.org>
Co-authored-by: beafialho <beafialho@git.wordpress.org>
Co-authored-by: markhowellsmead <markhowellsmead@git.wordpress.org>
Co-authored-by: youknowriad <youknowriad@git.wordpress.org>
Co-authored-by: annezazu <annezazu@git.wordpress.org>
Co-authored-by: carolinan <poena@git.wordpress.org>
Co-authored-by: luminuu <luminuu@git.wordpress.org>
```
